_docs/7_SubsquidArchive/docker-compose.yml

```diff
@@ -0,0 +1,83 @@
+services:
+  db:
+    image: postgres:12
+    restart: always
+    volumes:
+      # replace VOLUME_NAME with your volume name
+      - /mnt/VOLUME_NAME/postgresql/data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: squid-archive
+
+  ingest:
+    depends_on:
+      - db
+    restart: on-failure
+    image: subsquid/substrate-ingest:firesquid
+    command: [
+      "-e", "ws://node:9944",
+      "-c", "10",
+       "--prom-port", "9090",
+       "--out", "postgres://postgres:postgres@db:5432/squid-archive"
+    ]
+    environment:
+      NODE_TLS_REJECT_UNAUTHORIZED: 0
+    ports:
+      - "9090:9090" # prometheus port
```

Comment:
Not sure that here and in other places we actually want to expose these ports on public interfaces.
Maybe for some, but likely not at all.